### DISCUSSION TITLE
Forked repo doesn't trigger action

### DISCUSSION THREAD

**Discussion by g1eny0ung** [community/community#25749] [Actions]
<p>I want to use this <a href="https://github.com/matchai/waka-box" rel="nofollow noopener">https://github.com/matchai/waka-box</a>, as following the README, I should fork the repo first.</p>
<p>But seems that the forked repo doesn’t trigger action.</p>
<p>So I want to know whether fork repo can trigger action?</p>
<p>Thx for reply.</p>

**Comment by ethomson** [comment]
<p>First, were you accepted to the beta for GitHub Actions?  If not, that’s a necessary first step.</p>
<p>Next, can you try editing the <code>.github/workflows/schedule.yml</code> and pushing that up to your repository?  The triggering mechanism may detect that.  There may be some limitations around detecting that workflow on your fork.</p>

**Comment by g1eny0ung** [comment]
<p>Yep, I 've accepted it.</p>
<p>I changed <code>schedule.yml</code> and push a new commit as you said, but it doesn’t work as before.</p>
<p>I also check part of the forks of <a href="https://github.com/matchai/waka-box" rel="nofollow noopener">https://github.com/matchai/waka-box</a>, seems that all forks didn’t trigger the GitHub action.</p>
<p>Is there anything wrong with the <code>waka-box</code> repo?</p>

**Comment by s-i-l-k-e** [comment]
<p>I’m also having this problem (on a different repo). Is this a bug with Github Actions?</p>

**Comment by imjohnbo** [comment, marked as answer]
<p>Hi there <a class="mention" href="/g1eny0ung">@g1eny0ung</a>! I was able to reproduce and fix with this temporary workaround. What about copying the contents of the <code>schedule.yml</code>, deleting the file, and creating a new file with the same name and contents as the original <code>schedule.yml</code>?</p>

**Comment by g1eny0ung** [comment]
<p><a class="mention" href="/imjohnbo">@imjohnbo</a> Thanks! It works for now!</p>

**Comment by unrevised6419** [comment]
<p>You have to delete file, commit, than recreate file and commit again?</p>

**Comment by g1eny0ung** [comment]
<p>Yep. I test it, only update can’t work.</p>

**Comment by j0k3r** [comment]
<p>Sadly this doesn’t work for us.</p>
<p>Creating a pull request from a fork still doesn’t trigger the GitHub Action for the CI :frowning:</p>

**Comment by FabianKoestring** [comment]
<p>Is this still not available?</p>

**Comment by VikasWanchoo** [comment]
<p>I am facng same issue.<br>
PR opened from forked repo doesn not trigger actions.</p>

**Comment by yordis** [comment]
<p>Hey there, any updates on this one <a class="mention" href="/imjohnbo">@imjohnbo</a> hopefully?</p>
<p>Same issue here, I forked a repo, I created a PR against my forked repo, and didn’t work. Even updated something in the workflow files just in case, and nothing.</p>

**Comment by omarcostahamido** [comment]
Hi @yordis et al,

Maybe you'd like to see my answer in #59603

Have a great day!

**Comment by jonathan-diazv** [comment]
I worked for me. Thanks
